README.md

```diff
@@ -30,7 +30,7 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>Hello Microverse Project</b></h3>
 
 </div>
```

Comment:
Good job so far setting up the readme, however, you have removed some important sections that can help a newbie get started which are:
- Tech stack
- Key features
- Install
- Run test
- Deployment
Kindly include this section according to this [example](https://github.com/ProgramKingAbel/Abel-Morara-Portfolio/tree/contact-form-validation) to make your project documentation more effective.


NOTE: Please remove the microverse logo from the readme or if you have an image, you can use it since microverse doesn't own this repo.